From ed64691764ec8323684f1c25d2f2a989ad77400f Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Thu, 27 Jun 2019 13:16:39 +0900
Subject: [PATCH 2/2] docs: update help message of SPP CLI

Command options of SPP CLI for getting IP address and port are updated
from getting separately, for example `-b 127.0.0.1` and `-a 7777`, to
get both with one option `-b 127.0.0.1:7777`. This patch is to update
the example of options.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 docs/guides/gsg/howto_use.rst | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/guides/gsg/howto_use.rst b/docs/guides/gsg/howto_use.rst
index c6023ee..3579c75 100644
--- a/docs/guides/gsg/howto_use.rst
+++ b/docs/guides/gsg/howto_use.rst
@@ -186,9 +186,8 @@ All of options can be referred with help option ``-h``.
     optional arguments:
       -h, --help            show this help message and exit
       -b BIND_ADDR, --bind-addr BIND_ADDR
-                            bind address, default=127.0.0.1
-      -a API_PORT, --api-port API_PORT
-                        bind address, default=777
+                            bind address, default=127.0.0.1:7777
+
 
 All of SPP CLI commands are described in :doc:`../../commands/index`.
 
-- 
2.17.1

